### PR TITLE
Update Invoke-SPOSiteSwap.md

### PR DESCRIPTION
**Pull request by Stealth7777**
Added an additional information block regarding permissions needed on the source site when running the page diag tool. Also changed "may need" to "will need" as it is required.

### CODE REVIEW

**Comment by yogkumgit** [comment]
@JohanFreelancer9

**Comment by yogkumgit** [comment]
@dariomws @tseward @get-itips

**Review by dariomws — APPROVED** [review]

File: sharepoint/sharepoint-ps/sharepoint-online/Invoke-SPOSiteSwap.md [hunk 1]
```diff
@@ -44,7 +44,7 @@ If the target is the root site at <https://tenant-name.sharepoint.com>, then the
...
 2. Functionality such as external sharing and application interfaces are dependent on the policies and permissions defined at the root site. Review the source site to ensure that it has the required policies and permissions as per the existing root site. This includes external sharing settings as well as site permissions.
-3. Larger tenants that have more than ~10,000 licenses may also need to run the [Page Diagnostic Tool](https://docs.microsoft.com/office365/enterprise/page-diagnostics-for-spo) against the source site. Any analysis results that have the category Attention required (Red) or Improvement opportunities (Orange) will need to be remediated before performing the swap.
+3. Larger tenants that have more than ~10,000 licenses will need to run the [Page Diagnostic Tool](https://docs.microsoft.com/office365/enterprise/page-diagnostics-for-spo) against the source site (Ensure that the user running the Page Diagnostic Tool has edit rights on the source site). Any analysis results that have the category Attention required (Red) or Improvement opportunities (Orange) will need to be remediated before performing the swap.
```
Comment (JohanFreelancer9) [inline sharepoint/sharepoint-ps/sharepoint-online/Invoke-SPOSiteSwap.md:47]:
```suggestion
3. Larger tenants that have more than ~10,000 licenses will need to run the [Page Diagnostic Tool](/office365/enterprise/page-diagnostics-for-spo) against the source site (ensure that the user running the Page Diagnostic Tool has edit rights on the source site). Any analysis results that have the category Attention required (Red) or Improvement opportunities (Orange) will need to be remediated before performing the swap. 
```

**Review by get-itips — APPROVED** [review]

**Review by tseward — APPROVED** [review]

**Comment by yogkumgit** [comment]
@tiburd Please review the copy comments and merge this PR. Thanks

**Pull request merged**